ReSharper.FSharp/test/src/FSharp.Tests/TypeProviders/RegexProviderTests.fs

```diff
@@ -0,0 +1,13 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Tests.Features.TypeProviders
+
+open JetBrains.ReSharper.TestFramework
+open NUnit.Framework
+
+[<TestReferences(Paths.RegexProviderDll); TestPackages("System.Text.RegularExpressions")>]
```

Comment:
There should be no need to commit its binary, I believe it should be possible to reference the provider package from NuGet. Does `[<TestPackages>]` work for you?